app/controllers/api/jobs_controller.rb

```diff
@@ -16,10 +25,53 @@ def info_for_output(jobs)
         url: job["url"],
         latitude: job["latitude"],
         longitude: job["longitude"],
-        key: job["jobkey"],
+        job_key: job["jobkey"],
         location: job["formattedLocationFull"],
         date_posted: job["formattedRelativeTime"]
       }
     end
   end
+
+  def job_params
+    params.require(:job).permit(:jobkeys)
+  end
+
+  def authorize!
```

Comment:
Is this only the responsibility of this controller? Are we going to require other controllers to be locked down in the future?